src/game.cpp

```diff
@@ -4466,6 +4483,12 @@ void Game::showOverlayMessage(const wchar_t *msg, float dtime,
 	delete[] msg;
 }
 
+void Game::showStatusTextSimple(const wchar_t *msg)
```

Comment:
if you do that (which is nice, just do wgettext inside it it ensure we translate ALL strings :)